CONTRIBUTING.md

```diff
@@ -2,27 +2,112 @@
 3. Submit a PR (don't review your own)!
     - To maintain a consistent style, we recommend running [Prettier](https://github.com/prettier/prettier) on js, and `mix format` before submission.
 4. Once a PR is merged, we can throw it up onto the dev server to see how things look!
 
+### Submitting a Pull Request
+
+For all but the absolute simplest changes, first find an existing GitHub issue or
+[submit a new issue](https://github.com/orcasound/orcasite/issues/new) so that the
+community can review and provide feedback. Getting early feedback will help ensure your work
+is accepted by the community. This will also allow us to better coordinate our efforts and
+minimize duplicated effort.
+
+If you would like to contribute, first identify the scale of what you would like to contribute.
+If it is small (grammar/spelling or a bug fix) feel free to start working on a fix. If you are
+submitting a feature or substantial code contribution, please discuss it with the maintainers and
+ensure it follows the public roadmap. You might also read these two blogs posts on contributing
+code: [Open Source Contribution Etiquette](http://tirania.org/blog/archive/2010/Dec-31.html) by Miguel de Icaza and
+[Don't "Push" Your Pull Requests](https://www.igvita.com/2011/12/19/dont-push-your-pull-requests/) by Ilya Grigorik.
+All code submissions will be reviewed and tested, and only those that meet
+the bar for both quality and design/roadmap appropriateness will be merged into the source.
+
+For all [pull requests](https://github.com/orcasound/orcasite/pulls) the following rules apply:
+- The pull request description should describe the problem and solution, and reference the GitHub issue if one exists.
+- Existing tests should continue to pass.
+- Tests should ideally be included for every bug or feature that is completed.
+- Documentation should be included for every feature that is end-user visible.
+- Coding style should be consistent with the style used in other files of the same type in this repository.
+
+## Contacting Us
+
 ### Questions?
+
+We are happy to answer as best we can any question you have about Orcasite.
+There are multiple ways you can ask questions:
+
+- Using the [Zulip channel](https://orcasound.zulipchat.com/#narrow/stream/437031-orcasite)
+- Using [Github discussions](https://github.com/orcasound/orcasite/discussions)
+
+### Meetings
+
+We meet for one hour every other Wednesday via Google Meet.  You can request a calendar invite or check the
+[Zulip channel](https://orcasound.zulipchat.com/#narrow/stream/437031-orcasite) for the latest meeting
+date and time information.
+
+- [Request a calendar invite](https://forms.gle/Tegj4x6qxWx7PSes5)
+- [Meeting link](https://meet.google.com/igp-gpdr-wwu)
+- [Meeting notes](https://docs.google.com/document/d/1zXngvGO5kdm24gqSTNfHleFifU6ldtbA4amGSXe5jpg/edit?tab=t.0#heading=h.qyef7i4h6hom)
```

Comment:
I would leave these out, don't want them to get indexed by crawlers (spam potential)